### PR TITLE
USWDS-Site: Fix component pages from audit (Phase 3)

### LINKED ISSUE #2151: USWDS-Site - Code audit: Fix inaccuracies in component pages (Phase 2)
Resolve issues found in audit findings:
[Code documentation audit - utilities pages](https://docs.google.com/spreadsheets/d/1YNid-FkaYHKJA7ezsJpCFhycACCpMPKmiNbo2QIf7lM/edit#gid=987418701) (Google sheets) :lock:

Breaking the fixes for component pages into phases to keep reviews manageable.

- [x] [Date range picker](https://designsystem.digital.gov/components/date-range-picker/)
- [x] [File input](https://designsystem.digital.gov/components/file-input/)
- [x] [Footer](https://designsystem.digital.gov/components/footer/)
- [x] [Form](https://designsystem.digital.gov/components/form/)
- [x] [Header](https://designsystem.digital.gov/components/header/)
- [x] [Icon list](https://designsystem.digital.gov/components/icon-list/)
- [x] [Icon](https://designsystem.digital.gov/components/icon/)
- [x] [Identifier](https://designsystem.digital.gov/components/identifier/)
- [x] [Input prefix/suffix](https://designsystem.digital.gov/components/input-prefix-suffix/)
- [x] [Language selector](https://designsystem.digital.gov/components/language-selector/)
- [x] [Link](https://designsystem.digital.gov/components/link/)
- [x] [List](https://designsystem.digital.gov/components/list/)
- [x] [Memorable date](https://designsystem.digital.gov/components/memorable-date/)
- [x] [Modal](https://designsystem.digital.gov/components/modal/)
- [x] [Pagination](https://designsystem.digital.gov/components/pagination/)

### LINKED ISSUE #2163: USWDS-Site - Code audit: Fix inaccuracies in component pages (Phase 3)
Resolve issues found in audit findings:
[Code documentation audit - utilities pages](https://docs.google.com/spreadsheets/d/1YNid-FkaYHKJA7ezsJpCFhycACCpMPKmiNbo2QIf7lM/edit#gid=987418701) (Google sheets) :lock:

Breaking the fixes for component pages into phases to keep reviews manageable.

- [ ] [Process list](https://designsystem.digital.gov/components/process-list/)
- [ ] [Prose](https://designsystem.digital.gov/components/prose/)
- [ ] [Radio buttons](https://designsystem.digital.gov/components/radio-buttons/)
- [ ] [Range slider](https://designsystem.digital.gov/components/range-slider/)
- [ ] [Search](https://designsystem.digital.gov/components/search/)
- [ ] [Select](https://designsystem.digital.gov/components/select/)
- [ ] [Side navigation](https://designsystem.digital.gov/components/side-navigation/)
- [ ] [Site alert](https://designsystem.digital.gov/components/site-alert/)
- [ ] [Step indicator](https://designsystem.digital.gov/components/step-indicator/)
- [ ] [Summary box](https://designsystem.digital.gov/components/summary-box/)
- [ ] [Table](https://designsystem.digital.gov/components/table/)
- [ ] [Tag](https://designsystem.digital.gov/components/tag/)
- [ ] [Text input](https://designsystem.digital.gov/components/text-input/)
- [ ] [Time picker](https://designsystem.digital.gov/components/time-picker/)
- [ ] [Tooltip](https://designsystem.digital.gov/components/tooltip/)
- [ ] [Typography](https://designsystem.digital.gov/components/typography/)
- [ ] [Validation](https://designsystem.digital.gov/components/validation/)

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary
Updated component pages according to the findings reported in the [code/developer content audit](https://docs.google.com/spreadsheets/d/1YNid-FkaYHKJA7ezsJpCFhycACCpMPKmiNbo2QIf7lM/edit#gid=987418701) (Google Docs :lock:)

> [!NOTE]
> For easier reviews, this PR includes only the simple changes. The changes that are included in this PR are marked in green on the spreadsheet. Items that need more discussion or their own issues are marked in yellow. Items marked in red will not be pursued. 

> [!Warning]
> Before merge, we should update all `NNNN-NN-NN` changelog dates.

## Related issue

Closes #2163 

## Preview link
- [Radio](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/radio-buttons/)
- [Site alert](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/site-alert/)
- [Table](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/table/)
- [Tag](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/tag/)
- [Text input](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/text-input/)

### CODE REVIEW

File: _components/text-input/text-input.md [hunk 1]
```diff
@@ -26,12 +26,12 @@ tags:
...
-  - variant: "`usa-input--error`"
+  - variant: "`.usa-input--error`"
     description: The error state of the text input.
-  - variant: "`usa-input--success`"
+  - variant: "`.usa-input--success`"
     description: The success state of the text input.
-  - variant: "`usa-input--[width]`"
-    description: Displays an input at a specific width. Accepts `2xs` (4ex), `xs` (7ex), `sm` or `small` (10ex), `md` or `medium` (20ex), `lg` (30ex), `xl` (40ex), and `2xl` (50ex).
+  - variant: "`.usa-input--[width]`"
+    description: Displays an input at a specific width. Accepts `2xs` (5ex), `xs` (9ex), `sm` or `small` (13ex), `md` or `medium` (20ex), `lg` (30ex), `xl` (40ex), and `2xl` (50ex).
```
Comment (mahoneycm) [inline _components/text-input/text-input.md:34]: Is this variant still used? I don't see any references to it in the design system and I'm unable to get it to change the width in Storybook 🤔

**Review by mahoneycm — COMMENTED** [review]
Looks good! Just one question about `usa-input--[width]` classes

### Testing checklist
- [x]  Radio
- [x]  Site alert
- [x]  Table
- [x]  Tag
- [ ]  Text input

File: _components/text-input/text-input.md (hunk 1 above)
Comment (mahoneycm) [reply _components/text-input/text-input.md:34]: Scratch that, I realized I had some conflicting styles!

**Review by mahoneycm — APPROVED** [review]
Looks good to me! All green changes are accounted for and accurate 👍 

### Testing checklist

- [x]  Radio
- [x]  Site alert
- [x]  Table
- [x]  Tag
- [x]  Text input

File: _components/radio-buttons/guidance/accessibility.md
Comment (mejiaj) [inline _components/radio-buttons/guidance/accessibility.md:1]: image

[Preview link](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/radio-buttons/#:~:text=Use%20proper%20labels%20and%20attributes.%20Each%20radio%20button%20should%20have%20a%20%3Clabel%3E.%20Associate%20the%20two%20by%20matching%20the%20%3Clabel%3E%E2%80%99s%20for%20attribute%20to%20the%20%3Cinput%3E%E2%80%99s%20id%20attribute.)

File: _components/site-alert/site-alert.md
Comment (mejiaj) [inline _components/site-alert/site-alert.md:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/site-alert/)

File: _components/table/guidance/implementation.md
Comment (mejiaj) [inline _components/table/guidance/implementation.md:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/table/#:~:text=Enable%20row%20sorting,following%20the%20table.)

File: _components/table/table.md
Comment (mejiaj) [inline _components/table/table.md:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/table/#:~:text=.usa%2Dtable%2D%2Dborderless)

File: _components/tag/tag.md
Comment (mejiaj) [inline _components/tag/tag.md:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/tag/#:~:text=has%20no%20settings.-,Tag%20variants,-Variant)

File: _components/text-input/text-input.md
Comment (mejiaj) [inline _components/text-input/text-input.md:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/text-input/#text-input-package:~:text=has%20no%20settings.-,Text%20input%20variants,-Variant)

File: _data/changelogs/component-text-input.yml
Comment (mejiaj) [inline _data/changelogs/component-text-input.yml:1]: image

[Preview](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-component-updates-phase-3/components/text-input/#text-input-package:~:text=Description-,NNNN%2DNN%2DNN,-N/A)

**Review by mejiaj — APPROVED** [review]
LGTM, added screenshots and preview links in relevant code files.

File: _data/changelogs/component-text-input.yml [hunk 2]
```diff
@@ -5,6 +5,11 @@ type: component
 changelogURL:
 
 items:
+  - date: NNNN-NN-NN
```
Comment (mejiaj) [inline _data/changelogs/component-text-input.yml:8]: @amyleadem can you update this field and merge when checks pass?

Comment (amyleadem) [reply _data/changelogs/component-text-input.yml]: Updated in [3ecbf7c](https://github.com/uswds/uswds-site/pull/2291/commits/3ecbf7cac1e1bb02dcd5b0399d26af246ce9cd88). Will merge when checks pass.

**Pull request merged**